src/widget/wtracktableview.h

```diff
@@ -28,10 +28,11 @@ class WTrackTableView : public WLibraryTableView {
   public:
     WTrackTableView(QWidget* parent, UserSettingsPointer pConfig,
                     TrackCollection* pTrackCollection, bool sorting = true);
-    ~WTrackTableView() override;
+    virtual ~WTrackTableView() override;
```

Comment:
'override' implies 'virtual', which is redundant here. I prefer to use either 'virtual' or 'override', but not both.